src/mongocxx/include/mongocxx/v1/encrypt.hpp

```diff
@@ -0,0 +1,46 @@
+// Copyright 2009-present MongoDB, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#pragma once
+
+#include <mongocxx/v1/encrypt-fwd.hpp>
+
+//
+
+#include <mongocxx/v1/detail/prelude.hpp>
+
+namespace mongocxx {
+namespace v1 {
+
+///
+/// Options related to field encryption for CSFLE and Queryable Encryption.
+///
+/// @see
+/// - [Fields and Encryption Types (MongoDB Manual)](https://www.mongodb.com/docs/manual/core/csfle/fundamentals/encryption-algorithms/)
+/// - [Encrypted Fields and Enabled Queries (MongoDB Manual)](https://mongodb.com/docs/manual/core/queryable-encryption/fundamentals/encrypt-and-query/)
+/// - [Encryption Schemas](https://www.mongodb.com/docs/manual/core/csfle/fundamentals/create-schema/)
+///
+/// @attention This feature is experimental! It is not ready for use!
+///
+class encrypt {};
```

Comment:
The "encrypt" options class is overloaded in usage with both the `.encrypt()` _database command_ as well as options pertaining to encrypted field configuration for CSFLE and Queryable Encryption. Unlike the `<command>_one` vs. `<command>_many` splits, this one was too complicated to attempt to split up by refined purpose as part of the v_noabi -> v1 migration effort. Perhaps it may be revisited in the future.